### ISSUE TITLE
pip install botocore: No matching distribution found for botocore==1.8.43

### ISSUE THREAD

**Issue by codrutartene** [boto/botocore#1386]
I have pip 9.0.1 (python 3.6) and I am trying to install gensim 3.3.0.

**Comment by andreineculau** [comment]
hitting the same issue now on travis with python 2.7

**Comment by racevedoo** [comment]
Same issue when installing awscli with python 2.7

**Comment by darabos** [comment]
Looks like 1.8.43 has been pushed incorrectly. It shows up on https://pypi.python.org/pypi/botocore/ but not on https://pypi.python.org/simple/botocore/.

**Comment by andreineculau** [comment]
friendly poke @JordonPhillips @kyleknap

**Comment by andreineculau** [comment]
related thread in aws-cli https://github.com/aws/aws-cli/issues/3149

**Comment by Pronjaboy** [comment]
Oh, same issue here. Thank god you guys are fast on the responses.

**Comment by gekitsuu** [comment]
Any word on when we can see the fix?

**Comment by hekaldama** [comment]
@darabos I am new to python package publishing, I followed https://pypi.python.org/simple/botocore/ and see references to release `1.8.43` on there now. Does it seem like it was resolved or is it still missing something?

**Comment by jamesls** [comment]
Are people still seeing this issue?  I'm not seeing issues when I try to install botocore.  It might have been a transient issue with pypi as nothing about our release process has changed lately.

```
$ pip install --no-cache-dir botocore==1.8.43
Collecting botocore==1.8.43
  Downloading botocore-1.8.43-py2.py3-none-any.whl (4.1MB)
    100% |████████████████████████████████| 4.1MB 22.3MB/s
Collecting docutils>=0.10 (from botocore==1.8.43)
  Downloading docutils-0.14-py2-none-any.whl (543kB)
    100% |████████████████████████████████| 552kB 5.3MB/s
Collecting jmespath<1.0.0,>=0.7.1 (from botocore==1.8.43)
  Downloading jmespath-0.9.3-py2.py3-none-any.whl
Collecting python-dateutil<3.0.0,>=2.1 (from botocore==1.8.43)
  Downloading python_dateutil-2.6.1-py2.py3-none-any.whl (194kB)
    100% |████████████████████████████████| 194kB 15.0MB/s
Collecting six>=1.5 (from python-dateutil<3.0.0,>=2.1->botocore==1.8.43)
  Downloading six-1.11.0-py2.py3-none-any.whl
Installing collected packages: docutils, jmespath, six, python-dateutil, botocore
Successfully installed botocore-1.8.43 docutils-0.14 jmespath-0.9.3 python-dateutil-2.6.1 six-1.11.0

$ pip --version
pip 9.0.1
```

**Comment by hekaldama** [comment]
@jamesls let me try again. My issue is from an EMR cluster trying to run bootstrap scripts which update to the latest release of `aws-cli`. It seems to be intermittent in testing throughout the last couple of hours.

**Comment by darabos** [comment]
Yeah, it appeared to be fixed shortly after I saw the error. 🙂 Sorry for the panic. Perhaps I should have reported that it was back in working order for us.

So it seems like this was just a "natural" delay somewhere? 1.8.43 was missing for at least a few hours...

**Comment by jamesls** [comment]
Hmm, not sure.  @dstufft any ideas?

**Comment by hekaldama** [comment]
@jamesls I have tested with > 4 clusters with _many_ nodes installing newest client and all are succeeding as they have in the past. This is resolved for me. Not sure on root cause and would be nice to know. Maybe we can get better `mtime`s on the uploads on pypi?

**Comment by codrutartene** [comment]
It is working well now!

**Comment by dstufft** [comment]
Yesterday the CDN provider on PyPI had a small disruption in the part of their service that handles ``PURGE`` requests. When the new version got uploaded to PyPI, and the PyPI backend software requested a ``PURGE`` of the currently cached entries, that purge didn't make it to every node in every PoP. This meant that some requests got served a stale response that didn't include the new release. This would have appeared to be an intermittent problem due to the nature of request routing within the CDN (you don't always get the same PoP or the same node within the PoP, and the differentiator between works or not is what PoP + node you were served by).

The disruption was quickly over, but it left any package that was uploaded in that time period in a potentially stale state. This would have fixed itself naturally over the course of a day or two as cached items fell out of the cache, however yesterday the PyPI admin team issued a manual purge to get everything back to 100%.
